### DISCUSSION TITLE
Etar on LineageOS -- view/uodate a Apple calendar on the Etar calendar - how ?

### DISCUSSION THREAD

**Discussion by LennyGL** [Etar-Group/Etar-Calendar#1736] [Q&A]
Not sure of the settings in the Etar calendar app(or Mac app) to view and update schedules added to a Mac calendar. I would like any changes made on my "Mac-mini" calendar to show up on my LineageOS and on phone calendar app. And updates or changes on my phone get updated on the Mac.

**Comment by jspricke** [comment]
That's out of scope of Etar, see our documentation here: https://github.com/Etar-Group/Etar-Calendar?tab=readme-ov-file#how-to-use-etar
